.eslintrc

```diff
@@ -32,7 +28,8 @@
         "import/consistent-type-specifier-style": ["error", "prefer-top-level"],
         "@typescript-eslint/prefer-ts-expect-error": "error",
         "@typescript-eslint/consistent-type-imports": ["error", {"prefer":  "type-imports", "fixStyle": "separate-type-imports"}],
-        "complexity": "off"
+        "complexity": "off",
+        "react/react-in-jsx-scope": "off"
```

Comment:
Guide also suggest `react/jsx-uses-react`

Reply:
We can simply extend `plugin:react/jsx-runtime` which does the job. No need to off the rules here.